### PR TITLE
fix(theme): revert navbar height

### PR DESCRIPTION
**Pull request by frassinier**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
- Navbar has its height set to `38px`
- Its search bar has no style

**What is the new behavior?**
- Navbar height has been reset to `48px`
- Align its search bar styles too (that markup is just used into theme demo AFAIK)

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:
- http://guidelines.talend.com/document/92132#/navigation-layout/app-header
- http://guidelines.talend.com/document/92132#/navigation-layout/search-bar

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://368.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://368.talend.surge.sh)

File: packages/theme/src/theme/_navbar.search.scss [hunk 1]
```diff
@@ -10,25 +10,20 @@
...
 			}
-
-			@media (min-width: $grid-float-breakpoint) {
-				padding-top: $navbar-padding-vertical;
-				padding-bottom: $navbar-padding-vertical;
-			}
 		}
 
 		.form-control {
-			padding-right: $navbar-search-btn-width;
```
Comment (ncomont) [inline packages/theme/src/theme/_navbar.search.scss]: `$navbar-search-btn-width` is not used anymore. Could you delete it from _variables.scss ?

File: packages/theme/src/theme/_navbar.search.scss [hunk 2]
```diff
@@ -10,25 +10,20 @@
 				content: '';
 				flex: 1 0 auto;
 			}
-
-			@media (min-width: $grid-float-breakpoint) {
-				padding-top: $navbar-padding-vertical;
```
Comment (ncomont) [inline packages/theme/src/theme/_navbar.search.scss]: `$navbar-padding-vertical` is not used anymore. Could you delete it from _variables.scss ?

File: packages/theme/src/theme/_navbar.search.scss [hunk 3]
```diff
@@ -10,25 +10,20 @@
...
-			min-height: $navbar-height;
 		}
 	}
 
 	.navbar-form {
 		.form-group {
-			padding: 0;
+			padding: $padding-small $padding-small $padding-smaller;
+			min-height: $navbar-height;
+			background: $white;
```
Comment (ncomont) [inline packages/theme/src/theme/_navbar.search.scss]: I think `$white` cause some issues on inverted navbar [see AppHEaderBar with simple form](http://368.talend.surge.sh/components/?selectedKind=AppHeaderBar&selectedStory=with%20simple%20form&full=0&down=1&left=1&panelRight=0&downPanel=kadirahq%2Fstorybook-addon-actions%2Factions-panel)

File: packages/theme/src/theme/_navbar.search.scss (hunk 2 above)
Comment (frassinier) [inline packages/theme/src/theme/_navbar.search.scss]: Yes, it is. In our components styles.

File: packages/theme/src/theme/_navbar.search.scss (hunk 1 above)
Comment (frassinier) [inline packages/theme/src/theme/_navbar.search.scss]: Yes, it is. In our components styles.

File: packages/theme/src/theme/_navbar.search.scss (hunk 3 above)
Comment (frassinier) [inline packages/theme/src/theme/_navbar.search.scss]: Good catch, fixed

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://368.talend.surge.sh)

File: packages/theme/src/theme/_navbar.search.scss (hunk 1 above)
Comment (frassinier) [inline packages/theme/src/theme/_navbar.search.scss]: You re right, I deleted it

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://368.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://368.talend.surge.sh)

**Pull request merged**